.github/workflows/self-test-and-release.yml

```diff
@@ -43,7 +43,14 @@ jobs:
       - name: Run Code Guru
         if: steps.iam-role.outcome == 'success'
         run: |
-          ./build/install/aws-codeguru-cli/bin/aws-codeguru-cli --region us-west-2 -r python-code --no-prompt -c "c5f9c93d5605bfe3a4de02896a26b7b91eb7c007:a574f23f6ea3fe607a6d33cc6c8fe28d81ab37a2"
+          ./build/install/aws-codeguru-cli/bin/aws-codeguru-cli --region us-west-2 -r python-code --no-prompt -c "5a0c9579b6c9b0ea7a5f5c1b9328bb7100c7a16e:a574f23f6ea3fe607a6d33cc6c8fe28d81ab37a2"
+
+      - name: Upload findings artifact
+        if: steps.iam-role.outcome == 'success'
+        uses: actions/upload-artifact@v1
+        with:
+          name: recommendations_python_${{ matrix.os }}
+          path: code-guru/
 
       - name: Print Summary of Findings Linux
```

Comment:
I suggest doing a case-insensitive Find for both Linux and Windows and matrixify all appearances.